fgpyo/sam/__init__.py

```diff
@@ -843,6 +843,21 @@ def all_recs(self) -> Iterator[AlignedSegment]:
             for rec in recs:
                 yield rec
 
+    def set_tag(
+        self,
+        tag: str,
+        value: Union[str, int, float],
```

Comment:
I think in pysam, if you want to clear a tag you set it to `None`, if that's right could we add `None` to the union too please?

Reply:
done